Interpreter/src/main/java/org/enso/interpreter/node/callable/MethodResolverNode.java

```diff
@@ -0,0 +1,75 @@
+package org.enso.interpreter.node.callable;
+
+import com.oracle.truffle.api.TruffleLanguage;
+import com.oracle.truffle.api.dsl.Cached;
+import com.oracle.truffle.api.dsl.CachedContext;
+import com.oracle.truffle.api.dsl.Specialization;
+import com.oracle.truffle.api.nodes.Node;
+import org.enso.interpreter.Language;
+import org.enso.interpreter.runtime.Context;
+import org.enso.interpreter.runtime.callable.DynamicSymbol;
+import org.enso.interpreter.runtime.callable.atom.Atom;
+import org.enso.interpreter.runtime.callable.atom.AtomConstructor;
+import org.enso.interpreter.runtime.callable.function.Function;
+import org.enso.interpreter.runtime.error.NoMethodErrorException;
+
+/**
+ * A node performing lookups of method definitions. Uses a polymorphic inline cache to ensure best
```

Comment:
```suggestion
 * A node performing lookups of method definitions. Uses a polymorphic inline cache to ensure the best
```

This may need the formatter run on it just due to line length.